packages/legend-graph/src/graphManager/AbstractPureGraphManager.ts

```diff
@@ -421,4 +422,12 @@ export abstract class AbstractPureGraphManager {
   abstract HACKY__extractServiceTestAssertionData(
     query: RawLambda,
   ): string | undefined;
+
+  // -------------------------------------- Mapping Analysis --------------------------------------
+
+  abstract doModelCoverageAnalytics(
+    graph: PureModel,
+    mapping: Mapping,
+    clientVersion: string,
+  ): Promise<ModelCoverageAnalyticsResult>;
 }
```

Comment:
I think we can expose a method here to build the mapping model coverage analytics result because I don't want `query` to call `V1_` method explicitly. The trick to do this is to come up with `type RawMappingModelCoverageResult = object` or so (see what we do for `RawLambda` for example)